<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gitl HEVC Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../icons/main.ico</normaloff>../../icons/main.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="imageWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="FrameView" name="imageView">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="sequencesListDockWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Sequence List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="SequenceList" name="sequenceListWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="pluginFilterListDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Plugin Filter List</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="PluginFilterList" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="msgDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Message Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="MsgViewer" name="msgTextBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="playbackDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Playback Control</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QSlider" name="progressBar">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>2</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="previousFrame">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextFrame">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="frame">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string> Frame</string>
              </property>
              <property name="margin">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="currentFrameNum">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1677215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string> 0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string> /</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="totalFrameNum">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>1677215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="margin">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="printScreenBtn">
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Snapshot</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBitstream">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_bitstream"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_bitstream_info_folder"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuBitstream"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_ori_yuv">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>open_ori_yuv</string>
   </property>
  </action>
  <action name="actionOpen_general">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>open_general</string>
   </property>
  </action>
  <action name="actionOpen_me">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>open_me</string>
   </property>
  </action>
  <action name="actionOpen_cu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>open_cu</string>
   </property>
  </action>
  <action name="actionOpen_pred">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>open_pred</string>
   </property>
  </action>
  <action name="actionOpen_bitstream">
   <property name="text">
    <string>open_bitstream</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Online Manual</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_benchmark_bitstream">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>open_benchmark_bitstream</string>
   </property>
  </action>
  <action name="actionHighlight_lcu">
   <property name="text">
    <string>highlight_lcu</string>
   </property>
  </action>
  <action name="actionOpen_bitstream_info_folder">
   <property name="text">
    <string>open_bitstream_info_folder(Advance)</string>
   </property>
  </action>
  <action name="actionOpen_benchmark_info_folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>open_benchmark_info_folder(Advance)</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameView</class>
   <extends>QGraphicsView</extends>
   <header>appgui/frameview.h</header>
  </customwidget>
  <customwidget>
   <class>MsgViewer</class>
   <extends>QTextBrowser</extends>
   <header location="global">appgui/msgviewer.h</header>
  </customwidget>
  <customwidget>
   <class>PluginFilterList</class>
   <extends>QListWidget</extends>
   <header>appgui/pluginfilterlist.h</header>
  </customwidget>
  <customwidget>
   <class>SequenceList</class>
   <extends>QListWidget</extends>
   <header>appgui/sequencelist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
